obfuscation/src/main/java/com/netflix/conductor/obfuscation/publisher/WorkflowObfuscationQueuePublisher.java

```diff
@@ -0,0 +1,62 @@
+package com.netflix.conductor.obfuscation.publisher;
+
+import com.netflix.conductor.common.metadata.workflow.WorkflowDef;
+import com.netflix.conductor.core.config.Configuration;
+import com.netflix.conductor.dao.QueueDAO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.List;
+
+@Singleton
+public class WorkflowObfuscationQueuePublisher {
```

Comment:
Should this be implementing the `WorkflowObfuscationQueuePublisher` in core module?